spec/rubocop/cop/root_cops/dont_mix_keyword_and_positional_args_spec.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "../../../spec_helper.rb"
+
+RSpec.describe RuboCop::Cop::RootCops::DontMixKeywordAndPositionalArgs do
```

Comment:
Cam we add specs for class methods:

```ruby
module SomeService
  def self.do_something(positional, kwarg:); end
end
```

Your offenses list only seems to include instance methods and that _seems_ unlikely to me